internal/render/render-template.subtests.go

```diff
@@ -0,0 +1,29 @@
+// +build go1.7
+
+package render
+
+import (
+	"io"
+
+	"github.com/cweill/gotests/internal/models"
+)
+
+func Header(w io.Writer, h *models.Header) error {
+	if err := tmpls.ExecuteTemplate(w, "header", h); err != nil {
+		return err
+	}
+	_, err := w.Write(h.Code)
+	return err
+}
+
+func TestFunction(w io.Writer, f *models.Function, printInputs bool) error {
```

Comment:
Instead of having two render files, actually you should add another parameter to TestFunction:

```go
func TestFunction(w io.Writer, f *models.Function, printInputs, useSubtests bool) error 
```

This parameter should be passed all the way from the `gotests` package, and the `main` file should determine this value. I think it should be enabled by default in Go 1.7, but be disable-able via a flag.